runtime/dali/src/governance.rs

```diff
@@ -0,0 +1,152 @@
+//! Runtime setup for the governance and democracy only.
+
+use super::*;
+use common::governance::native::*;
+use frame_support::traits::LockIdentifier;
+
+pub type NativeDemocracy = democracy::Instance1;
+
+pub type NativeCouncilMembership = membership::Instance1;
+pub type NativeTechnicalMembership = membership::Instance2;
+
+parameter_types! {
+	pub const CouncilMotionDuration: BlockNumber = 7 * DAYS;
+	pub const CouncilMaxProposals: u32 = 100;
+	pub const CouncilMaxMembers: u32 = 100;
+}
+
+impl membership::Config<NativeCouncilMembership> for Runtime {
+	type Event = Event;
+	type AddOrigin = EnsureRootOrHalfNativeCouncil;
+	type RemoveOrigin = EnsureRootOrHalfNativeCouncil;
+	type SwapOrigin = EnsureRootOrHalfNativeCouncil;
+	type ResetOrigin = EnsureRootOrHalfNativeCouncil;
+	type PrimeOrigin = EnsureRootOrHalfNativeCouncil;
+	type MembershipInitialized = Council;
+	type MembershipChanged = Council;
+	type MaxMembers = CouncilMaxMembers;
+	type WeightInfo = weights::membership::WeightInfo<Runtime>;
+}
+
+impl collective::Config<NativeCouncilCollective> for Runtime {
+	type Origin = Origin;
+	type Proposal = Call;
+	type Event = Event;
+	type MotionDuration = CouncilMotionDuration;
+	type MaxProposals = CouncilMaxProposals;
+	type MaxMembers = CouncilMaxMembers;
+	type DefaultVote = collective::PrimeDefaultVote;
+	type WeightInfo = weights::collective::WeightInfo<Runtime>;
+}
+
+impl membership::Config<NativeTechnicalMembership> for Runtime {
+	type Event = Event;
+	type AddOrigin = EnsureRootOrHalfNativeTechnical;
+	type RemoveOrigin = EnsureRootOrHalfNativeTechnical;
+	type SwapOrigin = EnsureRootOrHalfNativeTechnical;
+	type ResetOrigin = EnsureRootOrHalfNativeTechnical;
+	type PrimeOrigin = EnsureRootOrHalfNativeTechnical;
```

Comment:
Can we make it so that the Native council can set the technical council as well? That way we have a nice hierarchy.

Reply:
looking